docker-compose.yml

```diff
@@ -42,6 +42,7 @@ services:
       PS_LANGUAGE: en
       PS_COUNTRY: fr
       PS_DEV_MODE: 0
+    command: sh -c 'while ! mysql -hmysql -uroot -proot -e "status"; do sleep 1; done && /tmp/docker_run.sh'
```

Comment:
Can we add a comment to explain what's the difference with the basic waiting script?